### ISSUE TITLE
Odd ON Behavior - Note created, but Obsidian says 'file not found' [BUG] 

### ISSUE THREAD

**Issue by cboebel** [chrisgrieser/shimmering-obsidian#36]
**Describe the Bug**
Create a new note with ON
Obsidian comes to the foreground, note is created in the vault (and template contents are inserted, if a template is used)
Obsidian displays a file not found error, showing the proper path of the note that was just created, and does not transition to new note in the display. 
New note is listed in Obsidian file listing, and browsing to that note shows the expected contents.
Note: I'm sure I've done something dumb here.

**To Reproduce**
1. ON testing
2. Watch Obsidian respond

Advanced URI and MetaData Extractor are both installed and enabled. There are 5 json files in the metadata extractor folder.
MacOS Monterey, 12.0.1
Obsidian 0.12.19

**Debugging Log**
Alfred Debug Log:
```
[20:25:40.796] Shimmering Obsidian[Script Filter] Queuing argument '(null)'
[20:25:43.971] Shimmering Obsidian[Keyword] Processing complete
[20:25:43.973] Shimmering Obsidian[Keyword] Passing output 'testing9' to Conditional
[20:25:43.973] Shimmering Obsidian[Conditional] Processing complete
[20:25:43.974] Shimmering Obsidian[Conditional] Passing output 'testing9' to Replace
[20:25:43.974] Shimmering Obsidian[Replace] Processing complete
[20:25:43.975] Shimmering Obsidian[Replace] Passing output 'testing9' to Arg and Vars
[20:25:43.975] Shimmering Obsidian[Arg and Vars] Processing complete
[20:25:43.976] Shimmering Obsidian[Arg and Vars] Passing output 'testing9' to Run Script
[20:25:43.980] Shimmering Obsidian[Run Script] Processing complete
[20:25:43.981] Shimmering Obsidian[Run Script] Passing output 'This is a test!' to Write Text File
[20:25:43.981] Shimmering Obsidian[Write Text File] Processing complete
[20:25:43.982] Shimmering Obsidian[Write Text File] Passing output '/Users/cb/CBSynced/testing9.md' to Call External Trigger
[20:25:43.983] Shimmering Obsidian[External] Processing complete
[20:25:43.983] Shimmering Obsidian[External] Passing output '/Users/cb/CBSynced/testing9.md' to Run Script
[20:25:44.268] Shimmering Obsidian[Run Script] Processing complete
[20:25:44.269] Shimmering Obsidian[Run Script] Passing output '/Users/cb/CBSynced/testing9.md
' to Replace
[20:25:44.269] Shimmering Obsidian[Replace] Processing complete
[20:25:44.270] Shimmering Obsidian[Replace] Passing output '/Users/cb/CBSynced/testing9.md' to Run Script
[20:25:44.381] Shimmering Obsidian[Run Script] Processing complete
[20:25:44.390] Shimmering Obsidian[Run Script] Passing output '' to Run Script
[20:25:44.689] Shimmering Obsidian[Run Script] Processing complete
[20:25:44.690] Shimmering Obsidian[Run Script] Passing output '' to Conditional
[20:25:44.690] Shimmering Obsidian[Conditional] Processing complete
[20:25:44.691] Shimmering Obsidian[Conditional] Passing output '' to Run Script
```

**Checklist**
- [x] I have updated to the latest version of this workflow, the Advanced URI plugin, and the Metadata Extractor plugin.
- [x] I have read and checked the [Troubleshooting Documentation](https://github.com/chrisgrieser/shimmering-obsidian/blob/main/documentation/Troubleshooting.md).
- [x] I read the [Breaking Changes Documentation](https://github.com/chrisgrieser/shimmering-obsidian/blob/main/documentation/Breaking%20Changes.md) to make sure the unexpected behavior is not a breaking change.

**Comment by chrisgrieser** [comment]
> Note: I'm sure I've done something dumb here.

Nope, it's a minor bug an my side (saving the wrong variable). 🙈 

Fixed in 2.9.3 :)

**Comment by cboebel** [comment]
Wow! That was fast. Thank you very much, @chrisgrieser!
